recipe/build.sh

```diff
@@ -6,12 +6,14 @@ cd build_release
 export CXXFLAGS="-fPIC ${CXXFLAGS}"
 export CFLAGS="-fPIC ${CFLAGS}"
 
-cmake ..  \
+cmake ${CMAKE_ARGS} ..  \
```

Comment:
@isuruf, where does this come from?

Reply:
compiler package

Reply:
Got it. Thanks!

xref: https://github.com/conda-forge/ctng-compiler-activation-feedstock/pull/35